src/google-classroom-extractor/tests/mapping/assignments/test_assignments.py

```diff
@@ -0,0 +1,214 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from typing import Dict
+from datetime import datetime
+import pandas as pd
+from google_classroom_extractor.mapping.assignments import coursework_to_assignments_dfs
+from google_classroom_extractor.mapping.constants import (
+    SOURCE_SYSTEM,
+    ENTITY_STATUS_ACTIVE,
+)
+
+# unique value for each column in fixture
+COURSE_ID = "1"
+ID = "2"
+TITLE = "3"
+DESCRIPTION = "4"
+STATE = "5"
+ALTERNATE_LINK = "6"
+CREATION_TIME = "7"
+UPDATE_TIME = "8"
+MAX_POINTS = "9"
+DUEDATE_YEAR = "2000"
+DUEDATE_MONTH = "10"
+DUEDATE_DAY = "11"
+DUETIME_HOURS = "12"
+DUETIME_MINUTES = "13.0"  # API returns float strings
+WORK_TYPE = "14"
+SUBMISSION_MODIFICATION_MODE = "15"
+ASSIGNEE_MODE = "16"
+CREATOR_USER_ID = "17"
+SCHEDULED_TIME = "18"
+TOPIC_ID = "19"
+
+
+# mutates first dict
+def merged_dict(dict1: Dict, dict2: Dict) -> Dict:
+    dict1.update(dict2)
+    return dict1
+
+
+def test_mappings():
+    # arrange
+    coursework_df = pd.DataFrame(
+        {
+            "courseId": [COURSE_ID],
+            "id": [ID],
+            "title": [TITLE],
+            "description": [DESCRIPTION],
+            "state": [STATE],
+            "alternateLink": [ALTERNATE_LINK],
+            "creationTime": [CREATION_TIME],
+            "updateTime": [UPDATE_TIME],
+            "maxPoints": [MAX_POINTS],
+            "workType": [WORK_TYPE],
+            "submissionModificationMode": [SUBMISSION_MODIFICATION_MODE],
+            "assigneeMode": [ASSIGNEE_MODE],
+            "creatorUserId": [CREATOR_USER_ID],
+            "dueDate.year": [DUEDATE_YEAR],
+            "dueDate.month": [DUEDATE_MONTH],
+            "dueDate.day": [DUEDATE_DAY],
+            "dueTime.hours": [DUETIME_HOURS],
+            "dueTime.minutes": [DUETIME_MINUTES],
+            "scheduledTime": [SCHEDULED_TIME],
+            "topicId": [TOPIC_ID],
+        }
+    )
+
+    # act
+    assignments_dfs: Dict[str, pd.DataFrame] = coursework_to_assignments_dfs(
+        coursework_df
+    )
+
+    # assert
+    assert len(assignments_dfs) == 1
+    assignments_df: pd.DataFrame = assignments_dfs[COURSE_ID]
+    row_count, column_count = assignments_df.shape
+    assert row_count == 1
+    assert column_count == 12
```

Comment:
I'm OK with leaving this for now. It might be nice for us to look into using equivalents to NUnit's [SetUp] and related functions to structure something where each test can have a single assert. That way if the assert on line 80 fails, we still run the test that has the assert on line 81.

Reply:
Yes, I was thinking the same thing.  Wasted a good chunk of time trying to figure out the PyTest way. Let me know if you find anything.